README.md

```diff
@@ -49,12 +49,6 @@ To get a local copy up and running follow these simple example steps.
 - Twitter: [@JorumOdiemo](https://twitter.com/twitterhandle)
 - LinkedIn: [LinkedIn](https://www.linkedin.com/in/jorumodiemo/)
 
-👤 **Author2**
-
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
-
 ## 🤝 Contributing
 
```

Comment:
- [ ] **Required:** Kindly add the title of this project and update the description with more sentences. This would make your project more professional.